### DISCUSSION TITLE
请问支持vps服务器上安装吗

### DISCUSSION THREAD

**Discussion by cvpas** [messense/aliyundrive-webdav#349] [Q&A] [answered]
想知道项目支持vps服务器上安装吗？如果可以的话，想实现本地使用webdav客户端链接到服务器上。

**Comment by messense** [comment, marked as answer]
可以，不过不建议，会走 VPS 的流量下载。

**Reply by cvpas** [reply]
再请教一下，如果vps容量只有10G，那能否通过本地webdav客户端上传一个大于10G的文件到阿里云呢？

**Reply by messense** [reply]
基本不占用磁盘空间。

**Reply by cvpas** [reply]
那请问centos7 使用docker-compose安装的话，yml用主页那个可以吗？还是需要啥单独配置呢？
